### ISSUE TITLE
Having issues powering on Xbox, are certain settings required? 

### ISSUE THREAD

**Issue by dalanmiller** [OpenXbox/xbox-smartglass-core-node#21]
Hey there! I'm able to discover and interact with a currently powered on Xbox. However, when the Xbox is powered off, I'm unable to discover it or power it on with a given IP and `liveid`. I wasn't able to find anything in the OpenXbox documentation or other repos indicating what I should check. I've already set my Xbox to have the 'Instant-On' mode instead of the 'Eco' mode. 

Under the hood is this essentially a Wake On LAN packet?

**Comment by unknownskl** [comment]
The device is supposed to be turned on for the discovery to work.

The power on command can be buggy sometimes. Sometimes it works right away and sometimes you need to invoke the command 3-5 times to turn the console on. Under the hood it is different from the Wake-On-Lan packet.
See documentation here: https://openxbox.org/smartglass-documentation/simple_message.html#power-on-request

**Comment by dalanmiller** [comment]
Gotcha, I did some playing around last night and finally got it to work. I think I needed to be in this state:

1. Xbox had no power 
1. Xbox was plugged in 
1. At this point `powerOn` would not work
1. I turned on Xbox manually (the horror 😱) 
1. I turned off the Xbox
1. I then tested this package using ports `5050`, `3074`, and `4087` and found `5050` to work

I haven't yet tested if a stage three the `powerOn` call will work. I think there is a difference as to when the power brick is showing orange or showing white.
